src/storage/accessors/SparqlDataAccessor.ts

```diff
@@ -137,6 +137,21 @@ export class SparqlDataAccessor implements DataAccessor {
     return this.sendSparqlUpdate(this.sparqlInsert(name, metadata, parent, triples));
   }
 
+  public async writeMetadata(identifier: ResourceIdentifier, metadata: RepresentationMetadata): Promise<void> {
+    const { name, parent } = this.getRelatedNames(identifier);
+    const triples = metadata.quads();
+
+    const def = defaultGraph();
+    if (triples.some((triple): boolean => !def.equals(triple.graph))) {
+      throw new NotImplementedHttpError('Only triples in the default graph are supported.');
+    }
```

Comment:
We currently don't check this in the `writeDocument` function so no need to add it here as that would give us different behaviour. (Perhaps we should check it, but not relevant right now).

Reply:
I copied it from the `writeDocument` function from SparqlDataAccessor, that's why I added it there

Reply:
I meant that we don't check the metadata in the `writeDocument` function, only the data. So no need to check the metadata here if we also don't check that in `writeDocument`.